MetaMask/metamask-extension
# feat: add What's New ENV variable for development

## **Description**

Adds a new ENV var `DEV_WHATS_NEW_SHOW_ALL`. When enabled, it will show all allowed announcements/notifications that are allowed in the What's App mobile. 

Used for development purposes only 

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to .metamaskrc
2. Add `DEV_WHATS_NEW_SHOW_ALL=true`
3. Open MetaMask to see the allowed notifications
4. Remove variable from .metamaskrc when you're done using it

## **Screenshots/Recordings**

<img width="1201" alt="Screenshot 2023-12-21 at 8 03 22 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/e0d54d5d-3ec6-4887-bb17-8df0ed2fe27b">


### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
